openstudiocore/src/cli/main.cpp

```diff
@@ -165,11 +165,17 @@ extern "C" {
   #ifndef _WIN32
     void Init_console(void);
     void Init_dbm(void);
-    void Init_gdbm(void);
     void Init_pty(void);
-    void Init_readline(void);
     void Init_syslog(void);
   #endif
+
+// Currently only work on linux
+ #ifdef __linux__ 
```

Comment:
@tijcolem i thought this worked on windows but not linux?

Reply:
just changed this and commented gdbm and readline out.